### PR TITLE
Testing phase

### PR DESCRIPTION
**Pull request by vpchung**
thanks, bruh.

### CODE REVIEW

File: testing-phase-workflow.cwl [hunk 1]
```diff
@@ -95,7 +96,7 @@ steps:
     run: https://raw.githubusercontent.com/Sage-Bionetworks/ChallengeWorkflowTemplates/v3.2/cwl/annotate_submission.cwl
     in:
       - id: submissionid
-        source: "#get_submissionid/submissionid"
+        source: "#submissionId"
```
Comment (thomasyu888) [inline testing-phase-workflow.cwl:99]: Why this submission id?

**Review by thomasyu888 — APPROVED** [review]
LGTM. just one comment

File: testing-phase-workflow.cwl (hunk 1 above)
Comment (vpchung) [reply testing-phase-workflow.cwl:99]: whoops,`"#get_submissionid/submissionid"` was residual code from my test runs (i think?).  `"#submissionId` should be the original value when you first wrote this wf.

**Comment by vpchung** [comment]
Thanks!

**Pull request merged**